RepoZ.App.Win/i18n/de-DE.xaml

```diff
@@ -38,6 +38,7 @@ Alternativ, kann der Computer auch über das Menü oben rechts nach Repositories
 	<system:String x:Key="Open in Git Bash">In Git Bash öffnen</system:String>
 	<system:String x:Key="Open in Finder">In Finder öffnen</system:String>
 	<system:String x:Key="Open in Terminal">Im Terminal öffnen</system:String>
+	<system:String x:Key="Open in Sourcetree">Im Sourcetree öffnen</system:String>	
```

Comment:
Sorry to say but in German, we'd say "In" instead of "Im". I know, German is hard. That's because ... it actually is.

Reply:
By the way, it is okay for "Im Terminal" 🙃